docs/qase.md

```diff
@@ -0,0 +1,21 @@
+## [Qase-Reporter](../pkg/qase/README.md)
+## [Qase-Patch-Validation-Create](#qase-patch-validation-create)
+
+
+# Qase-Patch-Validation-Create
+
+### Description
+This job aims to automatically create a patch validation in Qase for a given project and test plan ID.
+
+### All needed variables are already defined or set in run time.
+
+- `QASE_API_TOKEN`: Qase API token to authenticate with the Qase API.
+- `QASE_PROJECT_CODE`: Qase project code where to create the patch validation.
```

Comment:
Should mention that its hardcoded to `K3SRKE2`

Reply:
its mentioned on jenkins...also locally if you want u can set another